tests/Unit/Lti/Extractor/ReplaceResultSourceIdExtractorTest.php

```diff
@@ -47,6 +47,20 @@ public function testItThrowsInvalidLtiReplaceResultBodyExceptionOnInvalidXmlCont
         $subject->extractSourceId('invalid');
     }
 
+    public function testItThrowsInvalidLtiReplaceResultBodyExceptionOnInvalidXmlNamespace(): void
+    {
+        $this->expectException(InvalidLtiReplaceResultBodyException::class);
+
+        $subject = new ReplaceResultSourceIdExtractor();
+
+        /** @var string $xmlContent */
+        $xmlContent = file_get_contents(
+            __DIR__ . '/../../../Resources/LtiOutcome/invalid_replace_result_body_wrong_namespace.xml'
```

Comment:
Please use `DIRECTORY_SEPARATOR` in order to work on windows. I think this XML, factory, etc should be inside taoLti extension, not spread in 3 places too.

https://github.com/oat-sa/extension-tao-outcomelti/pull/50#discussion_r483001591

Reply:
@gabrielfs7 -> taoLti extension is not a part of simple roaster ,
the formula with '/../' is used at line 36, 58 and 71 so in order to be consistent either we change all 3 places or we leave it like this
@Skyld what's your input